staging/src/kubevirt.io/client-go/examples/listvms/README.md

```diff
@@ -0,0 +1,23 @@
+# listvms
+
+List all VMs and VMIs objects in KubeVirt
+
+## How to build
+```
+$ export GO111MODULE=on
+$ go mod init
+$ go get k8s.io/apimachinery@kubernetes-1.14.0
+$ go build
+```
+
+## How to test this example
+first you need to export an environment variable `KUBECONFIG` pointing to your kubernetes config, where KubeVirt is installed.
+
+```
+$ export KUBECONFIG=/home/<user>/go/src/kubevirt.io/kubevirt/_ci-configs/k8s-1.13.3/.kubeconfig
```

Comment:
Since the goal is to have it copied to the staging repo, I would suggest to use something more generic:

```
KUBECONFIG=<nondefaultpath>
```
or just skip this line.